page/using-jquery-core/working-with-selections.md

```diff
@@ -5,16 +5,17 @@
 
 ### Getters & Setters
 
-jQuery "overloads" its methods, so the method used to set a value generally has the same name as the method used to get a value. When a method is used to set a value, it's called a setter. When a method is used to get (or read) a value, it's called a getter. Setters affect all elements in a selection. Getters get the requested value only for the first element in the selection.
+Some jQuery methods can be used to either assign or read some value on a selection. When the method is called with a value as an argument, it's referred to as a setter because it sets (or assigns) that value. When the method is called with no argument, it gets (or reads) the value of the element. Setters affect all elements in a selection, whereas getters return the requested value only for the first element in the selection, with the exception of [`.text()`](http://api.jquery.com/text/), which retrieves the values of all the elements.
 
 ```
-// The .html() method used as a setter:
+// The .html() method sets all the h1 elements' html to be "hello world":
 $( "h1" ).html( "hello world" );
 ```
 
 ```
-// The .html() method used as a getter:
-$( "h1" ).html();
+// The .html() method returns the html from the first h1 element:
+$( "h1" ).html(); // "hello world"
```

Comment:
We had a discussion on how we should display output of specific code in examples. On my phone right now but will dig that up tomorrow.